{{cookiecutter.project_slug}}/clients/vue3/vue.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   devServer: {
     proxy: {
       '/api/': {
-        target: (process.env.VUE_APP_DEV_SERVER_BACKEND +'/api/'|| 'https://{{ cookiecutter.project_slug }}-staging.herokuapp.com') + '/api/',
+        target: (process.env.VUE_APP_DEV_SERVER_BACKEND || 'http://server:8000'),
```

Comment:
`.env.local` already generated with a reference to pointing at staging.
Since there are now 3 ways to find the BE (staging, local, and docker), this change means we will use the env setting if it's set. If it isn't, we'll assume docker.